recovery_request.go

```diff
@@ -1,7 +1,4 @@
 package dbft
 
 // RecoveryRequest represents dBFT RecoveryRequest message.
-type RecoveryRequest interface {
-	// Timestamp returns this message's timestamp.
-	Timestamp() uint64
-}
+type RecoveryRequest interface{}
```

Comment:
`any`? It's somewhat strange then though.

Reply:
I've removed this commit, `Timestamp()` doesn't hurt here.